views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -126,3 +131,4 @@ Cypress.Commands.add('startTest', (testName) => {
 
     cy.wait(['@testRunnerInit', '@testRunnerGetItem']);
```

Comment:
this line seems like is not needed, and if you remove it then the route @testRunnerInit is completely unnecessary

Reply:
The purpose of this line is wait til you have an item in the DOM.
I was thinking in a way that if we don't have an `init test request` and `get me the first item request` we cannot say that we are inside a test.
I can remove it, if you think its better.

Reply:
Maybe it is just my env, but sometimes it works, sometimes does not.